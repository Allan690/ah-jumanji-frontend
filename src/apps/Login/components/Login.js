import React from "react";

const Login = () => (
  <div className="login">
    <h3>Login</h3>
    <label htmlFor="email">
      Email
      <input id="email" name="email" type="email" placeholder="Email Address" />
    </label>
    <br />
    <button type="button">Submit</button>
  </div>
);

export default Login;
