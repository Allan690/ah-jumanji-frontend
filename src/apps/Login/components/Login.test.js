import React from 'react';
import {shallow} from 'enzyme';
import Login from './Login';

  describe('<Login /> snapshot', () => {
    it('should match the snapshot', () => {
      const component = shallow(<Login debug />);
      expect(component).toMatchSnapshot();
    });
  });