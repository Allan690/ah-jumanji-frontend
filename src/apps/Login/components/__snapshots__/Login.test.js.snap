// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> snapshot should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoginForm
    debug={true}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="panel panel-default"
        id="login-form"
      >
        <div
          className="panel-body"
        >
          <form>
            <button
              className="btn btn-success btn-block"
              type="submit"
            >
              Submit
            </button>
            <Connect(SocialAuthActions) />
            <a
              href="/forgotpassword"
            >
              Forgot Password?
            </a>
            <br />
            Don't have an account?
            <a
              href="/register"
            >
              Register.
            </a>
          </form>
        </div>
      </div>,
      "className": "col-md-6",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="panel-body"
        >
          <form>
            <button
              className="btn btn-success btn-block"
              type="submit"
            >
              Submit
            </button>
            <Connect(SocialAuthActions) />
            <a
              href="/forgotpassword"
            >
              Forgot Password?
            </a>
            <br />
            Don't have an account?
            <a
              href="/register"
            >
              Register.
            </a>
          </form>
        </div>,
        "className": "panel panel-default",
        "id": "login-form",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <form>
            <button
              className="btn btn-success btn-block"
              type="submit"
            >
              Submit
            </button>
            <Connect(SocialAuthActions) />
            <a
              href="/forgotpassword"
            >
              Forgot Password?
            </a>
            <br />
            Don't have an account?
            <a
              href="/register"
            >
              Register.
            </a>
          </form>,
          "className": "panel-body",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <button
                className="btn btn-success btn-block"
                type="submit"
              >
                Submit
              </button>,
              <Connect(SocialAuthActions) />,
              <a
                href="/forgotpassword"
              >
                Forgot Password?
              </a>,
              <br />,
              "Don't have an account?",
              <a
                href="/register"
              >
                Register.
              </a>,
            ],
            "onSubmit": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Submit",
                "className": "btn btn-success btn-block",
                "type": "submit",
              },
              "ref": null,
              "rendered": "Submit",
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Forgot Password?",
                "href": "/forgotpassword",
              },
              "ref": null,
              "rendered": "Forgot Password?",
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "br",
            },
            "Don't have an account?",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Register.",
                "href": "/register",
              },
              "ref": null,
              "rendered": "Register.",
              "type": "a",
            },
          ],
          "type": "form",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="panel panel-default"
          id="login-form"
        >
          <div
            className="panel-body"
          >
            <form>
              <button
                className="btn btn-success btn-block"
                type="submit"
              >
                Submit
              </button>
              <Connect(SocialAuthActions) />
              <a
                href="/forgotpassword"
              >
                Forgot Password?
              </a>
              <br />
              Don't have an account?
              <a
                href="/register"
              >
                Register.
              </a>
            </form>
          </div>
        </div>,
        "className": "col-md-6",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="panel-body"
          >
            <form>
              <button
                className="btn btn-success btn-block"
                type="submit"
              >
                Submit
              </button>
              <Connect(SocialAuthActions) />
              <a
                href="/forgotpassword"
              >
                Forgot Password?
              </a>
              <br />
              Don't have an account?
              <a
                href="/register"
              >
                Register.
              </a>
            </form>
          </div>,
          "className": "panel panel-default",
          "id": "login-form",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <form>
              <button
                className="btn btn-success btn-block"
                type="submit"
              >
                Submit
              </button>
              <Connect(SocialAuthActions) />
              <a
                href="/forgotpassword"
              >
                Forgot Password?
              </a>
              <br />
              Don't have an account?
              <a
                href="/register"
              >
                Register.
              </a>
            </form>,
            "className": "panel-body",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <button
                  className="btn btn-success btn-block"
                  type="submit"
                >
                  Submit
                </button>,
                <Connect(SocialAuthActions) />,
                <a
                  href="/forgotpassword"
                >
                  Forgot Password?
                </a>,
                <br />,
                "Don't have an account?",
                <a
                  href="/register"
                >
                  Register.
                </a>,
              ],
              "onSubmit": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Submit",
                  "className": "btn btn-success btn-block",
                  "type": "submit",
                },
                "ref": null,
                "rendered": "Submit",
                "type": "button",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Forgot Password?",
                  "href": "/forgotpassword",
                },
                "ref": null,
                "rendered": "Forgot Password?",
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "Don't have an account?",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Register.",
                  "href": "/register",
                },
                "ref": null,
                "rendered": "Register.",
                "type": "a",
              },
            ],
            "type": "form",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
